tests/Backend/Exasol/TableDefinitionOperationsTest.php

```diff
@@ -350,7 +350,76 @@ public function testRemovePrimaryKeyOnTypedTable()
 
     public function testCreateSnapshotOnTypedTable()
     {
-        $this->expectExceptionMessage("Not implemented for typed tables");
-        $this->_client->createTableSnapshot($this->tableId);
+        $bucketId = $this->getTestBucketId(self::STAGE_IN);
+
+        $snapshotId = $this->_client->createTableSnapshot($this->tableId, 'table definition snapshot');
+
+        $newTableId = $this->_client->createTableFromSnapshot($bucketId, $snapshotId, 'restored');
+        $newTable = $this->_client->getTable($newTableId);
+        $this->assertEquals('restored', $newTable['name']);
+
+        $this->assertSame(['id'], $newTable['primaryKey']);
+        $this->assertSame('HASH', $newTable['distributionType']);
```

Comment:
predpokladam, ze to je copy paste ze synapse bez upravy, protoze EXA DB dostupna. Prosim tedy o doplneni

Reply:
jasně sorry sem ten test neměl na čem pustit, teď exasol už jede tak ho dopracuju.

Reply:
doplnil sem